lib/elements/satin_column.py

```diff
@@ -898,6 +914,21 @@ def _get_split_points(self, left, right, max_stitch_length, count=None):
             points.append(Point(split_point.x, split_point.y))
         return [points, split_count]
 
+    def _do_short_stitches(self, sides):
+        for i, (left, right) in enumerate(zip(*sides)):
+            if i % 2 == 0:
+                continue
+            if left.distance(sides[0][i-1]) < self.short_stitch_distance:
+                split_point = self._get_split_point(left, right, self.short_stitch_inset)
+                sides[0][i] = Point(split_point.x, split_point.y)
+            if right.distance(sides[1][i-1]) < self.short_stitch_distance:
+                split_point = self._get_split_point(right, left, self.short_stitch_inset)
+                sides[1][i] = Point(split_point.x, split_point.y)
+
+    def _get_split_point(self, point1, point2, distance_fraction):
+        line = shgeo.LineString([point1, point2])
+        return line.interpolate(distance_fraction, normalized=True)
```

Comment:
Cool trick:

`return point1 * (1 - distance_fraction) + point2 * distance_fraction`

Reply:
And as an added bonus, that returns a Point object.